### PR TITLE
v4.3.1

### PR DESCRIPTION
**Pull request by Antonior26**
* Fixed problems with preparation method and cancer participant in Clinical Report

Added Participants 1.0.4 with the following changes:
CancerParticipant
* Add yearOfBirth
* remove TNMStage/Grouping fields
* Add morphology field

TumourSample
* Rename tumourType -> diseaseType
* Rename tumourSubType -> diseaseSubType
* Rename phase -> tumourType
* Add 3 morphology fields
* Add 3 topography fields

### CODE REVIEW

**Comment by Antonior26** [comment]
Weird it worked for me....

**Comment by Antonior26** [comment]
OH I see it now :)

**Comment by greglever** [comment]
yeah `org.gel.models.participant.avro::1.0.3-SNAPSHOT` is still there in `build.py`

File: build.py [hunk 1]
```diff
@@ -180,13 +168,15 @@ def main():
         nargs='+',
         default=[
             'org.gel.models.participant.avro::1.0.0', 'org.gel.models.participant.avro::1.0.1',
-            'org.gel.models.participant.avro::1.0.2',
+            'org.gel.models.participant.avro::1.0.2', 'org.gel.models.participant.avro::1.0.3-SNAPSHOT',
```
Comment (greglever) [inline build.py]: `org.gel.models.participant.avro::1.0.3-SNAPSHOT` should be changed to `org.gel.models.participant.avro::1.0.3` and `org.gel.models.participant.avro::1.0.4-SNAPSHOT` needs to be added

**Pull request closed without merging**